src/components/clusterWizard/wizardTransition.ts

```diff
@@ -210,7 +213,12 @@ export const getWizardStepClusterStatus = (
 ): Cluster['status'] => {
   const { status } = cluster;
   if (['insufficient', 'pending-for-input'].includes(cluster.status)) {
-    const validationsInfo = stringToJSON<ClusterValidationsInfo>(cluster.validationsInfo) || {};
+    const validationsInfo = stringToJSON<ClusterValidationsInfo>(cluster.validationsInfo);
+    if (!validationsInfo) {
```

Comment:
Same as with host functions. This is not needed as check functions will return false if validationsInfo is empty.